install/tor-snowflake-install.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: KernelSailor
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://snowflake.torproject.org/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  ca-certificates \
+  curl \
+  git \
+  jq
+msg_ok "Installed Dependencies"
+
+setup_go
+
+msg_info "Creating snowflake user"
+useradd -r -s /bin/false -d /opt/snowflake snowflake
+msg_ok "Created snowflake user"
+
+msg_info "Building Snowflake Proxy from Source"
+cd /opt || exit
+RELEASE=$(curl -fsSL https://gitlab.torproject.org/api/v4/projects/tpo%2Fanti-censorship%2Fpluggable-transports%2Fsnowflake/releases | jq -r '.[0].tag_name' | sed 's/^v//')
+$STD git clone https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports/snowflake.git
+cd snowflake || exit
```

Comment:
```suggestion
cd /opt
RELEASE=$(curl -fsSL https://gitlab.torproject.org/api/v4/projects/tpo%2Fanti-censorship%2Fpluggable-transports%2Fsnowflake/releases | jq -r '.[0].tag_name' | sed 's/^v//')
$STD git clone https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports/snowflake.git
cd snowflake
```

We don't really like git clone, are there tagged releases you could fetch or is there a github mirror?